### PR TITLE
added Dockerfile

### LINKED ISSUE #36: Dockerise dashboard
### Steps

- [ ] Create Dockerfile for the Streamlit dashboard.
- [ ] Test the containerised application to ensure all components work together seamlessly.
- [ ] Optimise Docker images for efficient deployment.

### User Story

As a developer, I want to containerise the application so that it can be easily deployed and scaled.

### LINKED ISSUE #37: Upload Dashboard Docker image to ECR
### Steps

- [x] Set up AWS ECR repositories for storing Docker images.
- [x] Push Docker images to ECR.
- [x] Configure access permissions for ECR.

### User Story

As a developer, I want to store container images in a centralised registry so that they can be easily accessed and deployed.

### PR DESCRIPTION
**Pull request by A-Faris**
Added Dockerfile so streamlit can be run from the cloud
closes #36 #37

### CODE REVIEW

File: Dashboard/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+RUN pip install -r requirements.txt
+
+COPY login.py .
+COPY navigation.py .
+COPY .streamlit/config.toml .
+COPY utilities/email_verification.py .
+COPY utilities/ses_get_emails.py .
+COPY pages/price.py .
+
+EXPOSE 80
```
Comment (BerkayDur) [inline Dashboard/Dockerfile]: Shouldnt this be `8501`?

File: Dashboard/Dockerfile
Comment (BerkayDur) [inline Dashboard/Dockerfile:1]: This Dockerfile doesn't follow streamlit's recommendations:
https://docs.streamlit.io/deploy/tutorials/docker

**Review by BerkayDur — CHANGES_REQUESTED** [review]

File: Dashboard/Dockerfile
Comment (A-Faris) [reply Dashboard/Dockerfile:1]: done

**Review by BerkayDur — APPROVED** [review]
Looks good

**Pull request merged**